src/saved_game.cpp

```diff
@@ -296,47 +296,40 @@ void saved_game::expand_scenario()
 
 void saved_game::check_require_scenario()
 {
-	if(starting_point_["addon_id"].empty()) {
-		return;
-	}
-
-	config required_scenario;
-
-	required_scenario["id"] = starting_point_["addon_id"];
-	required_scenario["name"] = starting_point_["addon_title"];
-	required_scenario["version"] = starting_point_["addon_version"];
-	required_scenario["min_version"] = starting_point_["addon_min_version"];
-	required_scenario["required"] = starting_point_["require_scenario"].to_bool(false);
-	config& content = required_scenario.add_child("content");
+	config scenario;
+	scenario["id"] = starting_point_["addon_id"].str("mainline");
+	scenario["name"] = starting_point_["addon_title"].str("mainline");
+	scenario["version"] = starting_point_["addon_version"];
```

Comment:
Perhaps this should default to the mainline/wesnoth version? (Same thing a little further down.)

Reply:
See my answer to gfgtdf just above.

Reply:
If that's what gfgtdf said then I agree.

Reply:
gfgtdf said he was wondering whether wesnoth's version should be used, so if you agree with that, then... should I add it? (not trying to be difficult; I'd just like something definitive one way or the other)

Reply:
I would definitely do it that way.

Reply:
Alright, updated.